test/manual/testplan.md

```diff
@@ -1,87 +1,200 @@
+| 1  | create a folder                                                               | 1. Add an account to desktop client<br>2. Open local sync folder<br>3. Create a folder<br>4. Wait for sync to complete                                                                                                                                                                                                                                                                                   | The folder is available on the server                                                                                                                                                           | :robot: Win<br>:construction: macOS<br>:robot: Linux               | tst_syncing                   |
+| 2  | folder with long name                                                         | 1. Create a folder with a long name (~100 characters)<br>2. Wait for sync to complete                                                                                                                                                                                                                                                                                                                    | The folder is available on the server                                                                                                                                                           | :robot: Win<br>:construction: macOS<br>:robot: Linux               | tst_syncing                   |
+| 3  | folder with special characters                                                | 1. Create a folder with special characters (e.g. `$%ñ&`)<br>2. Wait for sync to complete                                                                                                                                                                                                                                                                                                                 | The folder is available on the server                                                                                                                                                           | :robot: Win<br>:construction: macOS<br>:robot: Linux               | tst_syncing                   |
```

Comment:
We should probably also test non ASCII characters.

Reply:
I've added some emojis